dts/bindings/power-domain/intel,adsp-power-domain.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Power domains for intel ADSP
+
+compatible: "intel,adsp-power-domain"
+
+include: power-domain.yaml
+
+properties:
+    lps_handle:
```

Comment:
lps_handle is just a register, wouldn't be easier / simpler having it directly here ?

Reply:
Do you mean in the `reg` property of this node?

Reply:
That's directly addressing @andyross comment. All of the power domains need access to the same registers, just different bits in it. Also, in terms of structure, it resides in low power sequencer, so I think that's the best option.